sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -967,6 +969,27 @@ def dead_letter_source(self):
                 pass
         return None
 
+    @property
+    def message_state(self):
+        # type: () -> Optional[str]
```

Comment:
should it not be optional? as it always returns a status (defaults to `ACTIVE`)

or .NET returns None if it's undefined?

```suggestion
        # type: () -> ServiceBusMessageState
```